modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/util/ConfigurationInstanceAware.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.extension.internal.util;
+
+import org.mule.runtime.core.api.Event;
+import org.mule.runtime.extension.api.runtime.ConfigurationInstance;
+
+import java.util.Optional;
+import java.util.function.Function;
+
+/**
+ * Objects who want to be aware of the {@link ConfigurationInstance} should implement this interface.
+ *
+ * @since 4.0
+ */
+public interface ConfigurationInstanceAware {
```

Comment:
wasn't the convention to name PepeAware the interface that provides setPepe(Pepe pepito)? This does not "locates" the instance.